update_application.sh

```diff
@@ -30,5 +39,8 @@ fi
 cp $REMOTE_SCRIPT_PATH/secrets.js ./
 npm install #--production
 #npm prune --production
+npm run build
+npm rebuild node-sass
 
-pm2 start $REMOTE_SCRIPT_PATH/ecosystem.config.json
+pm2 start $REMOTE_SCRIPT_PATH/ecosystem.config.json --env production
+service ngnix start
```

Comment:
Are you talking about restarting nginx or the production flag to pm2? I'm removing both: 1) you're right, its not necessary to restart nginx and 2) we're specifying NODE_ENV=production elsewhere and for some reason, passing the --env production variable here breaks deployment. Setting the --env production variable adds NODE_ENV=production to the call pm2 makes to start the application.